Mergin/project_dialog.py

```diff
@@ -0,0 +1,42 @@
+import os
+from qgis.PyQt.QtWidgets import QDialog, QDialogButtonBox, QFileDialog, QApplication, QMessageBox
+from qgis.PyQt import uic
+
+ui_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'ui', 'ui_create_project.ui')
+
+
+class ProjectDialog(QDialog):
```

Comment:
Please fix the vertical spacing for clone project (and also for create project) - the content should stay centered, not pushed to the bottom...